draft-ietf-taps-interface.md

```diff
@@ -2354,11 +2354,14 @@ that no bound is specified.
 
 ## TCP-specific Property: User Timeout Option (UTO) {#tcp-uto}
```

Comment:
Since we're now using plural, should this be "Properties"?

Reply:
Yes, I guess so.